content/en/platform/corda/4.11/enterprise/node/collaborative-recovery/collaborative-recovery.md

```diff
@@ -0,0 +1,25 @@
+---
+date: '2023-11-20'
+menu:
+  corda-enterprise-4-11:
+    identifier: corda-enterprise-4-11-corda-nodes-collaborative-recovery
+    name: "Collaborative Recovery"
+    parent: corda-enterprise-4-11-corda-nodes
+tags:
+- disaster recovery
+- collaborative recovery
+- install
+- node operator
+
+title: Collaborative Recovery
+weight: 100
+---
+
+# Collaborative Recovery
+
+The [Ledger Recovery flow]({{< relref "ledger-recovery/ledger-recovery-flow.md" >}}) (`LedgerRecoveryFlow`) was introduced
+as part of the Corda 4.11 release. It enables a node to identify and recover transactions from peer recovery nodes to which
+it was a party and which are missing from its own ledger. Use it to re-instate a Corda database if your version of Corda is 4.11 and above.
+
+If you are using a pre-4.11 Corda, use [Collaborative Recovery V1.2.1]({{< relref "collaborative-recovery-121/introduction-cr.md" >}})
```

Comment:
This is the 4.11 version of the docs. Seems strange to be addressing users of a different version.

Reply:
Yes, I thought the same but Marcello told me that there is a reason for leaving it in case customers who are used to  Collaborative Recovery V1.2.1 still want to use it in 4.11 and above. I updated the wording slightly to let the users know that they can use both in 4.11 and above.